src/firebase-import.js

```diff
@@ -80,29 +82,74 @@ function promptToContinue(ref, next) {
   }
 }
 
+function getJsonFromFile(file, callback) {
+  var firstChar;
+  var rs = fs.createReadStream(file);
+  rs.on('data', function(chunk) {
+    var s = chunk.toString().trim();
+    if (s !== "") {
+      rs.close();
```

Comment:
I wasn't sure about Promises because I assumed this needed to support node versions below 0.12.